docs/management/updates.md

```diff
@@ -30,6 +30,31 @@ If your version is lower than `8.2`, it will not receive updates anymore. To kee
 
 Your dom0 system must either have access to the internet, or to a local mirror. In the second case, make sure to update the `baseurl` values in `/etc/yum.repos.d/xcp-ng.repo` to make them point at the local mirror, and keep the mirror regularly synced.
 
+#### Proxy configuration
+
+If your hosts need to use a proxy to be able to access the repositories, there are multiple ways to configure it depending on your use case:
+
+For all yum repositories:
+- Set `proxy=<proxy_url>` in `[main]` section of `/etc/yum.conf`
+
+For specific repositories:
+- Set `proxy=<proxy_url>` in the repository section of your choice in files present in `/etc/yum.repos.d`
+
+Set a system-wide proxy that will be used for everything, not only yum:
+- Add you can export lines for `http_proxy` and `https_proxy` variables in `/etc/environment`
+```bash
+export http_proxy=<proxy_url>
+export https_proxy=<proxy_url>
+```
+
+:::warning
+There are caveats for each methods:
```

Comment:
Other caveats: none of this sticks after an upgrade to a newer version, AFAIK.

Reply:
I added a line in this callout to mention it (see last fixup).